vaccine_feed_ingest/runners/in/arcgis/normalize.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+# import schema
+site_dir = pathlib.Path(__file__).parent
+state_dir = site_dir.parent
+runner_dir = state_dir.parent
+root_dir = runner_dir.parent
+sys.path.append(str(root_dir))
+
+from schema import schema  # noqa: E402
+
+# Configure logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("ak/arcgis/normalize.py")
+
+
+def _get_id(site: dict) -> str:
+    data_id = site["attributes"]["GlobalId"]
+
+    # Could parse these from directory traversal, but do not for now to avoid
+    # accidental mutation.
+    site = "arcgis"
+    runner = "in"
+
+    # Could parse these from the input file name, but do not for now to avoid
+    # accidental mutation.
+    arcgis = "46630b2520ce44a68a9f42f8343d3518"
+    layer = 0
+
+    return f"{runner}:{site}:{arcgis}_{layer}:{data_id}"
+
+
+def _get_inventory(site: dict) -> Optional[List[schema.Vaccine]]:
+    vaccines_field = site["attributes"]["Vaccine_Type"].lower().split(",")
+
+    potentials = {
```

Comment:
Can you use the vaccine name constants specified on this wiki page? https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Normalized-Location-Schema

Thank you!

Reply:
Sorry about that - these changed last night to match another system, and I forgot to update my review here to include the new values. My bad!